packages/suite/test/health/urls.test.ts

```diff
@@ -15,9 +14,8 @@ const getUrls = () => {
     const constantUrls = Object.values(URLS).filter((url: string) => {
         return !excluded.includes(url);
     });
-    const externalNetworksUrls = externalNetworks.map(n => n.url);
 
-    return [...constantUrls, ...externalNetworksUrls];
+    return constantUrls;
```

Comment:
I think we can return the urls directly rather than assigning it to a variable. In fact, we can push it further, by omitting all explicit return statements, make it a variable instead of a function and even make it a one-liner if prettier doesn't bark at us :dog: 
```
const filteredUrls = Object.values(URLS).filter((url: string) => !excluded.includes(url));
```